camel/agents/chat_agent.py

```diff
@@ -943,8 +949,31 @@ def _step_model_response(
         str,
     ]:
         r"""Internal function for agent step model response."""
+
+        response = None
         # Obtain the model's response
-        response = self.model_backend.run(openai_messages)
+        for _ in range(len(self.model_backend.models)):
```

Comment:
Please help me understand, why would we run all the models for every message?

Reply:
Added break statement for fixing this bug.